irr/include/IEventReceiver.h

```diff
@@ -331,16 +331,19 @@ struct SEvent
 
 		//! A bitmap of button states. You can use isButtonPressed() to determine
 		//! if a button is pressed or not.
-		//! Currently only valid if the event was EMIE_MOUSE_MOVED
+		//! Currently only valid if the event was EMIE_MOUSE_MOVED or EMIE_MOUSE_WHEEL
```

Comment:
at least the sdl device sets it for the other two events (pressed down, left up) too

Reply:
Alright, checked all the devices:

- [x] SDL2 device
- [x] Linux device
- [x] macOS device: There were two paths where it might not have initialized `ButtonStates`. This should be fixed now.
- [x] Win32 device

Removed the comments now, since it should always be valid now: https://github.com/minetest/minetest/pull/14702/commits/565a9a74e3c0659c0eb1433303a58056e20a79ce